client/src/productOverview/overviewApp.jsx

```diff
@@ -36,15 +37,19 @@ const OverviewApp = (props) => {
       {(overviewProduct === null || allStyles === null || currentStyle === null)
         ? <div>Loading</div>
         :
-        <div className="overview-container">
+        <div className="overview-container" id="overview-container">
           <div className="image-gallary">
-            <Image productStyle={currentStyle}/>
-          </div>
-          <div className="style-section">
-            <Rating reviewCount={props.reviewCount} rating={props.rating}/>
-            <Title product={overviewProduct}/>
-            <Style currentStyle={currentStyle} setCurrentStyle={setCurrentStyle} allStyles={allStyles} />
+            <Image currentStyle={currentStyle} expendView={expendView} setView={setView}/>
           </div>
+          {expendView ?
```

Comment:
I think you mean "expandView". expand is to grow, where expend is to do use or spend money.